### ISSUE TITLE
Taking tribute of syndicate is too easy

### ISSUE THREAD

**Issue by Oeleo** [endless-sky/endless-sky#313]
I took tribute of republic planets and it was really hard, even with >10 bactrians with Quarg stuff (and I liked that) but seriously... Syndicates make me feel bad about them again, they just sent me beetween 10 and 20 ships, I finished all their planet's in 1 hour lol, wasn't they supposed to be riched enough to make ships ? Even some pirate's planet were harder. And an other thing, republic ships are **far** stronger than Syndicat's ships, maybe they could buy ship to the Navy with their huge amount of money or there could be a way to give them some good ship, but we need to do something for them :,(

**Comment by endless-sky** [comment]
Most of the Syndicate worlds are actually intended to be rather "poor" - sparsely populated mining worlds, etc. The tribute payments should always be proportional to the fleet strengths, so a handful of Paradise worlds will give you more income than all the Syndicate worlds combined.

**Comment by Oeleo** [comment]
Oh ok I thought that they was supposed to be rich, that's why I found this paradoxical. I found it little equilibrate because Syndicates had't **one** planet which was a challenge, maybe that's because they havn't some good ship like the carrier (in addition to their little number of course). I think that with 3 bactrians with normal weapons it is possible to take tribute most of their planet. Republic planet with tribute between 1000 and 2000 was certainly harder to take. And even if most of the planet are populated mining, why isn't there one or few of the planet which have a good fleet ? The planets where the richiest men than we met in the Free world quest live ? They probably bought a lot of ships to protect themselves.
